src/components/suggest/suggest.tsx

```diff
@@ -221,9 +221,22 @@ export const EuiSuggest: FunctionComponent<EuiSuggestProps> = ({
       'euiSuggest.stateUnsaved',
       'euiSuggest.stateUnchanged',
     ],
-    ['State: loading', 'State: saved', 'State: unsaved', 'State: unchanged']
+    ['State: loading.', 'State: saved.', 'State: unsaved.', 'State: unchanged.']
   );
 
+  const stateMessage = (() => {
+    switch (status) {
+      case 'loading':
+        return stateLoading;
+      case 'saved':
+        return stateSaved;
+      case 'unsaved':
+        return stateUnsaved;
+      case 'unchanged':
+        return stateUnchanged;
+    }
+  })();
```

Comment:
def not a required suggestion, just curious if you think a map feels more readable that an immediately invoked function switch?

```suggestion
const stateMessageMap = {
  loading: stateLoading,
  saved: stateSaved,
  unsaved: stateUnsaved,
  unchanged: stateUnchanged,
};
const stateMessage = stateMessageMap[status];
```